packages/perseus/src/widgets/interactive-graphs/graphs/polygon.tsx

```diff
@@ -360,3 +498,69 @@ export const hasFocusVisible = (
         return matches(":focus");
     }
 };
+
+function getPolygonGraphDescription(
+    state: PolygonGraphState,
+    i18n: I18nContextType,
+    markings: InteractiveGraphProps["markings"],
+): string {
+    const strings = describePolygonGraph(state, i18n, markings);
+    return strings.srPolygonInteractiveElements;
+}
+
+type PolygonGraphDescriptionStrings = {
+    srPolygonGraph: string;
+    srPolygonGraphPointsNum: string;
+    srPolygonGraphPoints?: string;
+    srPolygonElementsNum: string;
+    srPolygonInteractiveElements: string;
+};
+
+function describePolygonGraph(
```

Comment:
I didn't export this one for testing, because I tested all these strings inside the LimitedPolygon component since it's called in there.